chandra_aca/plot.py

```diff
@@ -184,61 +196,83 @@ def plot_stars(attitude, catalog=None, stars=None, title=None, starcat_time=None
                       to be selected as acq stars.  If None, bad_stars will be set by a call
                       to bad_acq_stars().
+    :param plot_keepout: plot CCD area to be avoided in star selection (default=False)
     :returns: matplotlib figure
     """
     if stars is None:
         quat = Quaternion.Quat(attitude)
         stars = agasc.get_agasc_cone(quat.ra, quat.dec,
                                      radius=1.5,
                                      date=starcat_time)
+
     if bad_stars is None:
         bad_stars = bad_acq_stars(stars)
 
     fig = plt.figure(figsize=(5.325, 5.325))
+    fig.subplots_adjust(top=0.95)
+
+    # Make an empty plot in row, col space
     ax = fig.add_subplot(1, 1, 1)
-    plt.subplots_adjust(top=0.95)
     ax.set_aspect('equal')
+    plt.xlim(-580, 590)  # Matches -2900, 2900 arcsec roughly
+    plt.ylim(-580, 590)
 
     # plot the box and set the labels
-    plt.xlim(2900, -2900)
-    plt.ylim(-2900, 2900)
-    b1hw = 2560
+    b1hw = 512
     box1 = plt.Rectangle((b1hw, -b1hw), -2 * b1hw, 2 * b1hw,
                          fill=False)
     ax.add_patch(box1)
-    b2w = 2600
+    b2w = 520
     box2 = plt.Rectangle((b2w, -b1hw), -4 + -2 * b2w, 2 * b1hw,
                          fill=False)
     ax.add_patch(box2)
 
-    ax.scatter([-2700, -2700, -2700, -2700, -2700],
-               [2400, 2100, 1800, 1500, 1200],
+    ax.scatter(np.array([-2700, -2700, -2700, -2700, -2700]) / -5,
+               np.array([2400, 2100, 1800, 1500, 1200]) / 5,
                c='orange', edgecolors='none',
                s=symsize(np.array([10.0, 9.0, 8.0, 7.0, 6.0])))
 
-    [l.set_rotation(90) for l in ax.get_yticklabels()]
-    ax.grid(grid)
-    ax.set_ylabel("Zag (arcsec)")
+    # Manually set ticks and grid to specified yag/zag values
+    yz_ticks = [-2000, -1000, 0, 1000, 2000]
+    zeros = [0, 0, 0, 0, 0]
+    r, c = yagzag_to_pixels(yz_ticks, zeros)
+    ax.set_xticks(r)
+    ax.set_xticklabels(yz_ticks)
+    r, c = yagzag_to_pixels(zeros, yz_ticks)
+    ax.set_yticks(c)
+    ax.set_yticklabels(yz_ticks)
+    ax.grid()
+
     ax.set_xlabel("Yag (arcsec)")
+    ax.set_ylabel("Zag (arcsec)")
+    [label.set_rotation(90) for label in ax.get_yticklabels()]
 
     if quad_bound:
-        pix_range = np.linspace(-510, 510, 50)
-        minus_half_pix = -0.5 * np.ones_like(pix_range)
-        # plot the row = -0.5 line
-        yag, zag = pixels_to_yagzag(minus_half_pix, pix_range)
-        ax.plot(yag, zag, color='magenta', alpha=.4)
-        # plot the col = -0.5 line
-        yag, zag = pixels_to_yagzag(pix_range, minus_half_pix)
-        ax.plot(yag, zag, color='magenta', alpha=.4)
-
-    # plot stars
+        ax.plot([-511, 511], [0, 0], color='magenta', alpha=0.4)
+        ax.plot([0, 0], [-511, 511], color='magenta', alpha=0.4)
+
+    if plot_keepout:
+        # Plot grey area showing effective keep-out zones for stars.  Back off on
```

Comment:
It's disabled by default, so we can just keep this code there and enable later if it seems useful.